ts/TextField/index.tsx

```diff
@@ -83,6 +87,8 @@ export class TextFieldInput extends MaterialComponent<
     'outlined'
   ];
 
+  protected mdcNotifyProps = ['valid', 'disabled'];
```

Comment:
Shouldn't `value` be added here? If the user choose to change the `value` prop it should notify mdc.

Reply:
Yes

Reply:
seems to be not necessary

Reply:
And causes issues

Reply:
Okay